lib/node_modules/@stdlib/blas/base/drot/README.md

```diff
@@ -0,0 +1,204 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# drot
+
+> Apply a plane rotation.
+
+<section class="intro">
+
+This BLAS level 1 routine applies a real plane rotation to real double-precision floating-point vectors. The plane rotation is applied to `N` points, where the points to be rotated are contained in vectors `x` and `y` and where the cosine and sine of the angle of rotation are `c` and `s`, respectively. The operation is as follows:
+
+<!-- <equation class="equation" label="eq:drot" align="center" raw="\begin{bmatrix}x_i \\ y_i\end{bmatrix} = \begin{bmatrix} c & s \\ -s & c\end{bmatrix}\begin{bmatrix} x_i \\ y_i \end{bmatrix}" alt="Plane rotation"> -->
+
+<!-- </equation> -->
+
+where `x_i` and `y_i` are the individual elements on which the rotation is applied.
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var drot = require( '@stdlib/blas/base/drot' );
+```
+
+#### drot( N, x, strideX, y, strideY, c, s )
+
+Applies a plane rotation.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+var y = new Float64Array( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+
+drot( x.length, x, 1, y, 1, 0.8, 0.6 );
+// x => <Float64Array>[ ~4.400, ~5.800, 7.2, 8.6, 10.0 ]
```

Comment:
Why the two trailing zeros? Should it just be `~4.4`? Meaning, are you wanting to convey an approximate value to one decimal point? Otherwise, I read `~4.400` as approximating, e.g., `4.40023456`.

Reply:
Yes it should be ~4.4 only, I missed that.

Reply:
It applies here and elsewhere throughout this PR.